New dbt cloud IA (#2932)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md

#### Configuring the application

The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

> To complete this section, you will need to create a login slug. This slug controls the URL where users on your account

Author: ```suggestion
To complete this section, you will need to create a login slug. This slug controls the URL where users on your account can log into your application. Login slugs are typically the lowercased name of your organization
```

> can log into your application. Login slugs are typically the lowercased name of your organization

Author: ```suggestion
```

separated with dashes. For example, the login slug for dbt Labs would be `dbt-labs`.
Login slugs must be unique across all dbt Cloud accounts, so pick a slug that uniquely identifies your company.

When prompted for the SAML 2.0 application configurations, supply the following values:

> - Single sign on URL: `https://cloud.getdbt.com/complete/saml`

Author: ```suggestion
* Single sign on URL: `https://YOUR_ACCESS_URL/complete/saml`
```

> - Audience URI (SP Entity ID): `https://cloud.getdbt.com/`

Author: ```suggestion
* Audience URI (SP Entity ID): `https://YOUR_ACCESS_URL/`
```

#### Configuring the application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

Author: ```suggestion
<Snippet src="access_url" />
```

### Configure the Okta application

> The following steps assume your dbt Cloud deployment uses`https://cloud.getdbt.com`. If your [deployment](/docs/cloud/about-cloud/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance. 

Author: ```suggestion
<Snippet src="access_url" />
```

> To complete this section, you will need to create a login slug. This slug controls the URL where users on your account

Author: ```suggestion
To complete this section, you will need to create a login slug. This slug controls the URL where users on your account can log into your application. Login slugs are typically the lowercased name of your organization
```

> can log into your application. Login slugs are typically the lowercased name of your organization

Author: ```suggestion
```

1. On the **SAML Settings** page, enter the following values:

>    * **Single sign on URL**: `https://cloud.getdbt.com/complete/okta`

Author: ```suggestion
   * **Single sign on URL**: `https://YOUR_ACCESS_URL/complete/okta`
```

>    * **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`

Author: ```suggestion
   * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`
```

### Configure the Google application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/cloud/about-cloud/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.

Author: ```suggestion
<Snippet src="access_url" />
```

2. Download the **IDP metadata**.
3. Copy the **SSO URL** and **Entity ID** and download the **Certificate** (or **SHA-256 fingerprint**, if needed).
4. Enter the following values on the **Service Provider Details** window:

>    - **ACS URL**: `https://cloud.getdbt.com/complete/saml`

Author: ```suggestion
   * **ACS URL**: `https://YOUR_ACCESS_URL/complete/saml`
```

>    - **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`

Author: ```suggestion
   * **Audience URI (SP Entity ID)**: `https://YOUR_ACCESS_URL/`
```

### Create Azure AD Enterprise application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/cloud/about-cloud/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.

Author: ```suggestion
<Snippet src="access_url" />
```

> To complete this section, you will need to create a login slug. This slug controls the URL where users on your account

Author: ```suggestion
To complete this section, you will need to create a login slug. This slug controls the URL where users on your account can log into your application. Login slugs are typically the lowercased name of your organization
```

> can log into your application. Login slugs are typically the lowercased name of your organization

Author: ```suggestion
```

   | Field | Value |
   | ----- | ----- |

>    | **Identifier (Entity ID)** | Base URL for dbt Cloud. Use `https://cloud.getdbt.com/`. |

Author: ```suggestion
   | **Identifier (Entity ID)** | Base URL for dbt Cloud. Use `https://YOUR_ACCESS_URL/`. |
```

>    | **Reply URL (Assertion Consumer Service URL)** | Use `https://cloud.getdbt.com/complete/saml`. |

Author: ```suggestion
   | **Reply URL (Assertion Consumer Service URL)** | Use `https://YOUR_ACCESS_URL/complete/saml`. |
```

### Configure the OneLogin application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/cloud/about-cloud/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.

Author: ```suggestion
<Snippet src="access_url" />
```

> To complete this section, you will need to create a login slug. This slug controls the URL where users on your account

Author: ```suggestion
To complete this section, you will need to create a login slug. This slug controls the URL where users on your account can log into your application. Login slugs are typically the lowercased name of your organization
```

> can log into your application. Login slugs are typically the lowercased name of your organization

Author: ```suggestion
```

3. Under the **Configuration tab**, input the following values:

   - **RelayState:** `<login slug>`

>    - **Audience (EntityID):** https://cloud.getdbt.com/

Author: ```suggestion
   - **Audience (EntityID):** https://YOUR_ACCESS_URL/
```

>    - **ACS (Consumer) URL Validator:** https://cloud.getdbt.com/complete/saml

Author: ```suggestion
   - **ACS (Consumer) URL Validator:** https://YOUR_ACCESS_URL/complete/saml
```

>    - **ACS (Consumer) URL:** https://cloud.getdbt.com/complete/saml

Author: ```suggestion
   - **ACS (Consumer) URL:** https://YOUR_ACCESS_URL/complete/saml
```

        title="Configuring the application in dbt Cloud" />

4. Click **Save** to complete setup for the SAML 2.0 integration.

> 5. After completing the setup, you can navigate to the URL generated for your account's _slug_ to

Author: ```suggestion
5. After completing the setup, you can navigate to the URL generated for your account's _slug_ to test logging in with your identity provider. Additionally, users added the the SAML 2.0 app will be able to log in to dbt Cloud from the IdP directly.
```

> test logging in with your identity provider. Additionally, users added the the SAML 2.0 app

Author: ```suggestion
```

> will be able to log in to dbt Cloud from the IdP directly.

Author: ```suggestion
```

> Users in your IdP will now be able to log into the application by navigating to the URL:

Author: ```suggestion
```

> `https://cloud.getdbt.com/enterprise-login/<login-slug>`

Author: ```suggestion
<Snippet src="login_url_note" />
```
